src/components/QueryExecutionStatus/QueryExecutionStatus.tsx

```diff
@@ -14,14 +14,14 @@ const b = cn('kv-query-execution-status');
 
 interface QueryExecutionStatusProps {
     className?: string;
-    error?: AxiosError | Record<string, any>;
+    error?: AxiosError | Record<string, any> | string;
```

Comment:
I didn't remove `Record<string, any>`, it seems the case with aborted error isn't covered by types yet. I plan to reconsider it later, when full components chain will be converted to TS (`Query` -> `QueryEditor` -> `ExecuteResult` && `ExplainResult`)